src/pages/City/CreateCity/CreateCity.tsx

```diff
@@ -68,7 +68,9 @@ const CreateCity = () => {
   const history = useHistory();
   const location = useLocation();
   const followerPath = "/regions/follower/";
-  const [loading, setLoading] = useState(false);
+  const isFollowerPath = location.pathname.includes(followerPath);
+
+  const [loaded, setLoaded] = useState<boolean>(false);
```

Comment:
If state are `boolean` it's a good practice to name it with `is` prefix
For example - `isLoaded`

Reply:
Ok